infrastructure/merklemountainrange/src/treemap_ser.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 The Tari Project
```

Comment:
Technically, this could go in a separate PR

Reply:
I could technically, but this was added here because it required it.